src/pywrangler/sync.py

```diff
@@ -46,6 +45,16 @@ def check_requirements_txt():
         raise click.exceptions.Exit(code=1)
 
 
+def check_wrangler_config():
+    wrangler_jsonc = PROJECT_ROOT / "wrangler.jsonc"
+    wrangler_toml = PROJECT_ROOT / "wrangler.toml"
+    if not wrangler_jsonc.is_file() and not wrangler_toml.is_file():
```

Comment:
Reduce negation complexity:
```suggestion
    if not (wrangler_jsonc.is_file() or wrangler_toml.is_file()):
```